main/main.go

```diff
@@ -80,25 +80,15 @@ func main() {
 	defer mapper.UnmapAllPorts()
 
 	// Open staking port
-	port, err := mapper.Map("TCP", Config.StakingLocalPort, stakingPortName)
-	if !Config.StakingIP.IsPrivate() {
-		if err == nil {
-			// The port was mapped and the ip is on a public network, the node
-			// should be able to be connected to peers on this public network.
-			Config.StakingIP.Port = port
-		} else {
-			// The port mapping errored, however it is possible the node is
-			// connected directly to a public network.
-			log.Warn("NAT traversal has failed. Unless the node is connected directly to a public network, the node will be able to connect to less nodes.")
-		}
-	} else {
-		// The reported IP is private, so this node will not be discoverable.
-		log.Warn("NAT traversal has failed. The node will be able to connect to less nodes.")
-	}
+	// we want for PnP to have the external port (Config.StakingIP.Port) to connect to our
+	// internal listening port (Config.InternalStakingPort) which should be the same in most cases.
+	mapper.Map("TCP", Config.InternalStakingPort, Config.StakingIP.Port, stakingPortName)
```

Comment:
Nit: here we specify that one port is "internal" and the other is the "regular" port, but for the HTTP port one is "external" and the other is "regular." Can be either do internal/internal or external/external for parallelism?

Reply:
I didn't want to rename variables to get this change.
Config.StakingIP.Port was always the external.  But renaming it would change a bunch of the code.  So I chose to leave it alone, but add Internal.

If you are aliment, I can rename the variables.

Reply:
Fair point...I think it's OK as is, then